### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Joisco** [microsoft/graphrag#366] [Q&A]
![Screenshot 2024-07-04 173345](https://github.com/microsoft/graphrag/assets/171664411/3684d74d-d5cb-4a9f-bde0-79de4d5b2cdf)
Not sure went wrong here, the log files are absolutely ginormous... anyone have any ideas?

**Comment by AlonsoGuevara** [comment]
Hi @Joisco 
Can you please attach your log file? I'll give it a read

**Reply by Joisco** [reply]
![image](https://github.com/microsoft/graphrag/assets/171664411/a44c804a-6095-4032-943b-5c781ab063dd)

how lol, its so big...

**Reply by AlonsoGuevara** [reply]
Woah!! Mind blown haha.
Mmmm... Maybe just the final bunch of lines with the stack trace?

**Reply by Joisco** [reply]
Will do soon, btw how big is the parquet file supposed to be? I used a 0.73 GB dataset, took about 4 hours to finish the 0.5 parquet file, and for some reason the log file is almost 16 GB.

**Reply by Joisco** [reply]
[Document.txt](https://github.com/user-attachments/files/16146837/Document.txt)

Have a crack at it, I don't know where to begin. Thanks for your assistance @AlonsoGuevara

**Reply by AlonsoGuevara** [reply]
Here's the issue:
```txt
'The model `gpt-4-turbo-preview` does not exist or you do not have access to it.', 'type': 'invalid_request_error', 'param': None, 'code': 'model_not_found'
```
So your entity extraction is failing due to model access. Can you please share your settings, if possible?

**Reply by Joisco** [reply]
[settings.txt](https://github.com/user-attachments/files/16183070/settings.txt)

I'm a complete beginner here, it looked like I need to fix a few things with this.
